cpanfile

```diff
@@ -50,7 +50,7 @@ requires 'JSON::Validator';
 requires 'JavaScript::Minifier::XS', '>= 0.11';
 requires 'LWP::Protocol::https';
 requires 'LWP::UserAgent';
-requires 'Minion', '>= 10.22';
+requires 'Minion', '>= 10.25';
 requires 'Minion::Backend::SQLite', '>= 5.0.1';
```

Comment:
from `perl -d -Ilib t/25-cache-service.t` I found out what seems to be the problem that in https://github.com/Grinnz/Minion-Backend-SQLite/blob/master/lib/Minion/Backend/SQLite.pm there is only

```
  my $stats = $self->sqlite->db->query(
    q{select
      (select count(*) from minion_jobs where state = 'inactive' and (expires is null or expires > datetime('now')))
        as inactive_jobs,
      (select count(*) from minion_jobs where state = 'active') as active_jobs,
      (select count(*) from minion_jobs where state = 'failed') as failed_jobs,
      (select count(*) from minion_jobs where state = 'finished') as finished_jobs,
      (select count(*) from minion_jobs where state = 'inactive' and delayed > datetime('now')) as delayed_jobs,
      (select count(*) from minion_locks where expires > datetime('now')) as active_locks,
      (select count(distinct(worker)) from minion_jobs where state = 'active') as active_workers,
      ifnull((select seq from sqlite_sequence where name = 'minion_jobs'), 0) as enqueued_jobs,
      (select count(*) from minion_workers) as inactive_workers,
      null as uptime}
  )->hash;
```

so missing the number of "registered workers" corresponding to the Pg backend in https://github.com/mojolicious/minion/commit/c15352a78c8ee2f24dcb4ede25b1884ec83a67de#diff-500e1fc27d944109f9aea9ca0c2d977016e5e2d4691a5ef2a90b04aaf529f254R238

Reply:
https://github.com/Grinnz/Minion-Backend-SQLite/pull/21